test/test_combined_inner_outer_reduction.cpp

```diff
@@ -242,15 +253,38 @@ TEST_F(NVFuserTest, CombinedSchedulerLayerNormBackward_CUDA) {
   std::vector<DataType> data_types = {DataType::Half, DataType::Float};
   std::vector<std::vector<int64_t>> batch_sizes = {{216}};
   std::vector<std::vector<int64_t>> hidden_sizes = {
-      {3}, {32}, {96}, {576}, {768}, {1024}, {1280}, {1600}, {1984}, {1987}};
+      {3},
+      {32},
+      {96},
+      {576},
+      {768},
+      {1024},
+      {1280},
+      {1600},
+      {1984},
+      {1987},
+      {27648}};
   bool isBenchmark = false;
   bool onlyTestFirstCase = false;
   int verbose = 0;
   for (auto dtype : data_types) {
     for (auto batch_shape : batch_sizes) {
       for (auto norm_shape : hidden_sizes) {
-        std::tuple<float, float> avg_var =
-            runTest(batch_shape, norm_shape, dtype, isBenchmark, verbose);
+        bool scale_down_to_avoid_overflow = false;
+        if (dtype == DataType::Half && norm_shape[0] == 27648) {
```

Comment:
Any problem if this is done always?

Reply:
It still works. But sacling down inputs seems like sacling up tolerence. It may lose the capability to capture some bugs leading to only a small difference in the results.

Reply:
How about scaling back the results?

Reply:
Good idea! If scale back, the scale factor only has a small influence the max diff.
left: scale_factor = 1, right: scale_factor = 0.01
![image](https://github.com/NVIDIA/Fuser/assets/116412316/581c71fd-e810-457c-a1e7-09e528976f48)

Reply:
Great idea!  If scale back, the max_diff matains at same level whether with or without scale.
Without scale:
```
max_diff= 1.90735e-06
max_diff= 1.90735e-06
max_diff= 0.000244141
max_diff= 0.00012207
max_diff= 0.00012207
max_diff= 0.00195312
max_diff= 0.000244141
max_diff= 0.000488281
max_diff= 0.000976562
max_diff= 0.00195312
max_diff= inf
```
Scale by 0.01 then scale back by 1/0.01
```
max_diff= 0
max_diff= 3.8147e-06
max_diff= 0.00012207
max_diff= 0.000976562
max_diff= 0.00012207
max_diff= 0.000488281
max_diff= 0.000976562
max_diff= 0.000488281
max_diff= 0.00390625
max_diff= 0.00195312
max_diff= 0.0078125
```